src/Shouldly/ShouldlyMessage.cs

```diff
@@ -240,20 +240,21 @@ public string CreateExpectedErrorMessage()
         private static string CreateActualVsExpectedMessage(IShouldlyAssertionContext context)
         {
             var codePart = context.CodePart;
-            string message = string.Format(@"
-    {0}
-        {1}
-    {2}
-        but was
-    {3}",
-                codePart, context.ShouldMethod.PascalToSpaced(), context.Expected.ToStringAwesomely(), context.Actual.ToStringAwesomely());
+            var message = string.Format(@"
+            {0}
+                {1}
+            {2}
+                but was
+            {3}",
+                codePart, context.ShouldMethod.PascalToSpaced(), context.Expected.ToStringAwesomely(),
+                context.ShouldMethod == "ShouldNotBe" ? string.Empty : context.Actual.ToStringAwesomely());
 
             if (DifferenceHighlighter.CanHighlightDifferences(context))
             {
                 message += string.Format(@"
-        difference
-    {0}",
-                DifferenceHighlighter.HighlightDifferences(context));
+                difference
```

Comment:
@JakeGinnivan thanks for the response. I think Resharper was overly keen to fix indentation. I should have known better. I will revert the space changes and push that in. Thanks